docs/sphinx/api/default_ops.rst

```diff
@@ -541,4 +541,74 @@ This is a non-linear transformation, and no template overloads are available.
     .. code-block:: cpp
 
         cudaq::qubit qubit;
-        my(qubit);
\ No newline at end of file
+        my(qubit);
+
+
+User-defined Custom Operations
```

Comment:
nit: I would vote for "defined" to be capitalized.
```suggestion
User-Defined Custom Operations
```

Reply:
Addressed in cc43fff33919ba6d1aa98985efc8e71863f48e62